src/pages/index.js

```diff
@@ -29,54 +32,153 @@ const enableValidation = (config) => {
 
 enableValidation(config);
 
-const createCard = (cardItem) => {
-  const card = new Card(cardItem, "#card-template", () => {
-    console.log(cardItem);
-    imgPopup.open(cardItem);
+//API
+
+export const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  authorization: "7dcd9a93-149c-4e56-87db-9285c9177a9e",
+});
+
+api
+  .getUserData()
+  .then((results) => results.json())
+  .then((data) => {
+    data;
+    currentAvatar = data.avatar;
+    initialUserData = new UserInfo(data);
   });
```

Comment:
You need to `catch` possible errors at the very **end** of any server request.

**Please, correct it everywhere**

Reply:
Corrected everywhere